tooling/ast_fuzzer/minimizer/scripts/minimize.sh

```diff
@@ -16,7 +16,25 @@ if [ -z "$CMD" ]; then
     usage "missing command"
 fi
 
-MAIN_PATH=$1; shift
+# Grab everything until we hit a .nr file. These are our compile options.
+OPTIONS=()
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        *.nr)
+            MAIN_PATH=$1
+            shift
+            break
+            ;;
+        *)
+            OPTIONS+=("$1")
+            shift
+            ;;
+    esac
+done
+
+# Build a string from the array of options
+OPTIONS="${OPTIONS[*]}"
```

Comment:
The people who came up with the syntax for stuff in Bash never cease to amaze me with their creativity. I don't know why it is that I can never remember any of it.